tests/conftest.py

```
@@ -0,0 +1,417 @@
+    "MARKER_LOAD_TIMEOUT",
+    "TABLE_LOAD_TIMEOUT",
+    "MOBILE_DEVICES",
+    "ALL_MOBILE_DEVICES",
+    "UI_VERTICAL_ALIGNMENT_TOLERANCE",
+    "TEST_LOCATIONS",
+    "webpack_script",
+    "page",
+    "window_open_stub",
+    "geolocation",
+    "performance_tracker",
+]
```

Ruff (0.14.11)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)